src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.PackTool.targets

```diff
@@ -18,7 +18,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     there is no dependency in nuspec. And add Microsoft.NETCore.Platforms, which is used to select correct RID assets.
     -->
   <PropertyGroup>
-    <NETCorePlatformsImplicitPackageVersion Condition=" '$(NETCorePlatformsImplicitPackageVersion)' == '' ">2.0.1</NETCorePlatformsImplicitPackageVersion>
+    <BundledNETCorePlatformsVersion Condition=" '$(BundledNETCorePlatformsVersion)' == '' ">2.0.1</BundledNETCorePlatformsVersion>
```

Comment:
The pattern should be that only CLI sets BundledXxxVersion.

In SDK, we should set XxxImplicitPackageVersion informed by BundledXxx.

Something like:
``` xml
<NETCorePlatformsImplicitPackageVersion Condition="'$(NETCorePlatformsImplicitPackageVersion)' == '' and '$(BundledNETCoreAppPlatformsVersion)' == ''">$(BundledNETCoreAppPlatformsVersion)</NETCorePlatformsImplicitPackageVersion>

Basically, the bundled versions don't change just because you want to use a different versions, they just say what version the CLI came with/knows about. Other variables in SDK default to them in various circumstances but can be overridden.

Reply:
Change to if NETCorePlatformsImplicitPackageVersion not set, use BundledNETCorePlatformsVersion, if still not set, then 2.0.1